userguide/system.rst

```diff
@@ -434,17 +434,16 @@ There are also other options available.
   in the :guilabel:`Active` column. Only alphanumeric characters,
   underscores, and dashes are allowed in the name.
 
-* **Scrub:** :guilabel:`Scrub Boot Pool` is used to perform a
-  manual scrub of the boot devices. By default, the boot device is
-  scrubbed every 7 days. To change the default interval, change the
-  number in the :guilabel:`Automatic scrub interval (in days)` field of
-  the :guilabel:`Boot Environments` screen. The date and results of the
-  last scrub are also listed in this screen. The condition of the boot
-  device should be listed as *HEALTHY*.
+* **Scrub Boot:** can be used to perform a manual scrub of the boot
+  devices. By default, the |os-device| is scrubbed every 7 days. To
+  change the default interval, change the number in the
+  :guilabel:`Automatic scrub interval (in days)` field. The date and
+  results of the last scrub are also listed in this screen. The
+  condition of the |os-device| should be listed as *HEALTHY*.
 
-* **Status:** click :guilabel:`Boot Pool Status` to see the status of
```

Comment:
The text in red is what should be present in this branch. Revert this change, with the addition of changing boot device to |os-device|.